exporters/prometheus/test/prometheus_test_helper.h

```diff
@@ -0,0 +1,237 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#ifndef ENABLE_METRICS_PREVIEW
+
+#  include "opentelemetry/version.h"
+
+namespace metric_sdk      = opentelemetry::sdk::metrics;
+namespace nostd           = opentelemetry::nostd;
+namespace exportermetrics = opentelemetry::exporter::metrics;
+
+/**
+ * Helper function to create ResourceMetrics
+ */
+metric_sdk::ResourceMetrics CreateSumPointData()
```

Comment:
Should we have all functions in this header as static inline? Also, later it would be good to move these functions at some common place, to be reusable for other exporters.

Reply:
thanks, made it inline.